tests/drivers/tee/optee/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+tests:
+  drivers.tee.optee:
+    build_only: false
+    tags: tee
+    timeout: 500
+    integration_platforms:
+      - native_posix_64
+    platform_allow: native_posix_64
```

Comment:
it would be nice to have this tests on a real boards and xen_dom0 shield

Reply:
please, see above